examples/clients/abe/examplepb_numeric_enum.go

```diff
@@ -0,0 +1,15 @@
+/* 
+ * examples/examplepb/a_bit_of_everything.proto
+ *
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * OpenAPI spec version: version not set
+ * 
+ * Generated by: https://github.com/swagger-api/swagger-codegen.git
+ */
+
+package abe
+
+// NumericEnum is one or zero.   - ZERO: ZERO means 0  - ONE: ONE means 1
+type ExamplepbNumericEnum struct {
```

Comment:
I don't know why. But this PR is neutral to the issue.
https://github.com/grpc-ecosystem/grpc-gateway/pull/415/files#diff-743714d76d1753794032b45e364fd780

Can we work on the issue in another PR?